core/domain/suggestion_services.py

```diff
@@ -216,10 +217,10 @@ def create_suggestion(
     suggestion_domain_class = (
         suggestion_registry.SUGGESTION_TYPES_TO_DOMAIN_CLASSES[
             suggestion_type])
-    # Here, suggestion_domain_class is of Union type which contains
-    # all suggestion classes, but out of these suggestion classes only
-    # `SuggestionEditStateContent` can accept Optional[str] value for
-    # language code and for other classes MyPy throws an `incompatible
+    # Here we use MyPy ignore because suggestion_domain_class is of Union
```

Comment:
I think this needs to be fixed properly and the types clarified using approaches we've talked about before.

Reply:
This issue is already fixed in https://github.com/oppia/oppia/pull/15958